xarray/tutorial.py

```diff
@@ -36,13 +36,48 @@ def _construct_cache_dir(path):
     "RGB.byte": "https://github.com/mapbox/rasterio/raw/1.2.1/tests/data/RGB.byte.tif",
     "shade": "https://github.com/mapbox/rasterio/raw/1.2.1/tests/data/shade.tif",
 }
+file_formats = {
+    "air_temperature": 3,
+    "rasm": 3,
+    "ROMS_example": 4,
+    "tiny": 3,
+    "eraint_uvz": 3,
+}
```

Comment:
should we also mention the backend in the list of the available datasets for `open_dataset`?

Reply:
This would be a good idea for future work :)